pages/content/amp-dev/documentation/index.html

```diff
@@ -0,0 +1,221 @@
+---
+$title: Get Started
+$titles:
+  navigation: Get started
+$view: /views/custom.j2
+$path: /documentation/index.html
+$localization:
+  path: /{locale}/documentation/index.html
+$order: 1
+$hidden: true
+flyout:
+  description: Get started
+  icon: start
+
+format_cards:
+  - title: AMP Websites
+    description: Create flawless wesbites. Compelling, smooth and instant.
+    type: websites
+    link:
+        text: Build your AMP Websites
```

Comment:
**Done**: I've changed the link titles to "Build your AMP Website / Story / Ad / Email" for now. I think that's what we should go with since we're using the Get Started Page to explain these four categories..?

Reply:
Jep, all the links should say something like "Build your first...".